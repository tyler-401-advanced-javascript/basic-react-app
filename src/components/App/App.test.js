import React from 'react'
import App from './'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

//this is why it fails when you change the snapshot
describe('<App />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('is alive at application start', () => {
    const component = shallow(<App />)
    expect(component.find('h1')).toExist()
  })
})
