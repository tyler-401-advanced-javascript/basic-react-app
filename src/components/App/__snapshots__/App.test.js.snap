// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    My App
  </h1>
  <section
    className="Counter"
  >
    <p>
      My wonderful component
    </p>
    <span>
      Stuff: 
      0
    </span>
    <button
      id="increase"
      onClick={[Function]}
    >
      Up
    </button>
    <button
      id="decrease"
      onClick={[Function]}
    >
      Down
    </button>
  </section>
</div>
`;
